question/engine/bank.php

```diff
@@ -419,8 +419,10 @@ public static function get_all_question_types_in_categories($categories) {
 
         list($categorysql, $params) = $DB->get_in_or_equal($categories);
         $sql = "SELECT DISTINCT q.qtype
-                FROM {question} q
-                WHERE q.category $categorysql";
+                           FROM {question} q
```

Comment:
indentation looks a bit weird, can you please double check

Reply:
Hi, I can see indented.

![image](https://user-images.githubusercontent.com/6785838/129496387-21bb72d0-11ac-437c-a6d9-9cc074c28930.png)